optuna/samplers/nsgaii/_sampler.py

```diff
@@ -119,6 +118,10 @@ def __init__(
         swapping_prob: float = 0.5,
         seed: int | None = None,
         constraints_func: Callable[[FrozenTrial], Sequence[float]] | None = None,
+        elite_population_selection_strategy: Callable[
```

Comment:
Could you add a docstring for this argument?

Reply:
Thanks for the comment. I added the explanation.